src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -1122,6 +1152,7 @@ public ListBoxModel doFillSshKeysCredentialsIdItems(@AncestorInPath ItemGroup co
             AbstractIdCredentialsListBoxModel result = new StandardListBoxModel();
             if (Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
                 result = result
+                        .includeEmptyValue()
```

Comment:
What if now a user select the doesn't select a value? How is configured the cloud? Should it have a FormValidation?

Reply:
it already does

Reply:
true, I just didn't connect this with the validation

```
                if (value == null || value.isEmpty()) {
                    return FormValidation.error("No ssh credentials selected");
                }
```